indico/migrations/versions/20220707_1007_b60f5c45acf7_add_option_to_enable_captcha.py

```diff
@@ -0,0 +1,25 @@
+"""Add option to registration forms to enable CAPTCHA for users without an account
```

Comment:
Way too verbose for a revision title ;)

"Add require_captcha to regforms" is shorter and more search-friendly since it contains the column name.